### PR TITLE
Revert "crio: drop infra container when possible"

### PR DESCRIPTION
**Pull request by deads2k**
This reverts commit 6eb37894974b71080648a92bbb71fd201c5a84f4.

reverts https://github.com/openshift/machine-config-operator/pull/2177, one of two commits that merged before the permafail of e2e-serial on OOMs

### CODE REVIEW

**Comment by haircommander** [comment]
/hold

for https://github.com/openshift/release/pull/13435 to merge so we can see if this revert fixes the serial tests

**Comment by deads2k** [comment]
> /hold
> 
> for [openshift/release#13435](https://github.com/openshift/release/pull/13435) to merge so we can see if this revert fixes the serial tests

It has merged.  If this goes green on aws-serial, we should merge this and plan the next attempt.

**Comment by haircommander** [comment]
/lgtm
/approve

(I am not sure I have approval rights though...)

**Review by yuqi-zhang — APPROVED** [review]
This will be gated on e2e-aws-serial anyways so we'll find out if this was the PR that broke it

**Comment by rphillips** [comment]
/hold cancel

... so this can potentially merge tonight

**Comment by haircommander** [comment]
/hold

I will keep an eye on it, I still wanna see if e2e-aws-serial passes or fails before merging

**Comment by haircommander** [comment]
this is the one

/hold cancel

**Pull request merged**
